Src/Plugins/PlaintextSecrets/PlaintextSecrets.json

```diff
@@ -4,15 +4,40 @@
       "Name": "DoNotExposePlaintextSecrets",
       "Id": "SEC1001",
       "Level": "Error",
+      "DefaultNameRegex": "(?i)\\.(azure|bat|cmd|config|cpp|cs|cscfg|definitions|dtsx|ini|java|jsx?|json|keys|loadtest|m|md|php|properties|ps1|psm1|pubxml|py|resx|sample|sql|ste|swift|test|tsx?|txt|waz|xml)$",
       "Description": "Do not expose plaintext (or base64-encoded plaintext) secrets in versioned engineering content.",
       "Message": "'{0:scanTarget}' contains an apparent {1:encoding}'{2:secretKind}' secret.",
       "MatchExpressions": [
         {
           "SubId": "AzureDevOpsPersonalAccessToken",
-          "NameRegex": "(?i)\\.(azure|bat|cmd|config|cpp|cs|cscfg|definitions|dtsx|ini|java|jsx?|json|keys|loadtest|m|md|php|properties|ps1|psm1|pubxml|py|resx|sample|sql|ste|swift|test|tsx?|txt|waz|xml)$",
           "Base64DecodedContentLength": 52,
           "ContentsRegex": "\\b(?<fingerprint>[a-z2-7]{52})",
           "MessageArguments": { "secretKind": "Azure DevOps personal access token" }
+        },
+        {
+          "SubId": "FacebookOAuthToken",
+          "ContentsRegex": "(?i)facebook.{0,30}['\"\\s]([0-9a-f]{32})['\"\\s]",
+          "MessageArguments": { "secretKind": "Facebook OAuth token" }
+        },
+        {
+          "SubId": "TwitterOAuthToken",
+          "ContentsRegex": "(?i)twitter.{0,30}['\"\\s]([0-9a-zA-Z]{35,44})['\"\\s]",
```

Comment:
>0-9a-zA-Z [](start = 56, length = 9)

We should test but I don't think this is required, because the entire match is case-insensitive due to (?i). Should be 0-9a-z? Definitely test if we change. :)